macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,278 @@
+meta:
+  id: mac_os_resource_snd
+  title: Classic MacOS Sound Resource
+  application: Sound Manager
+  endian: be
+  license: MIT
+  xref:
+    wikidata: Q7564684
+doc-ref: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
+doc: Sound resources were introduced in Classic MacOS with the Sound Manager program.
+  They can contain sound commands to generate sounds with given frequencies as well as sampled sound data.
+  They are mostly found in resource forks, but can occasionally appear standalone or embedded in other files.
```

Comment:
Thanks! However, please use [YAML multiline syntax](https://yaml-multiline.info/) instead:

```suggestion
doc: |
  Sound resources were introduced in Classic MacOS with the Sound Manager program.
  They can contain sound commands to generate sounds with given frequencies as well as sampled sound data.
  They are mostly found in resource forks, but can occasionally appear standalone or embedded in other files.
```

The reason is that when you use any of the [_flow scalars_](https://yaml-multiline.info/#flow-scalars) (plain, single-quoted or double-quoted), single newlines are converted to a single space (at least they _should be_ in a YAML-compliant parser, a quick test at https://ide.kaitai.io/devel/ shows that the [yaml.js](https://github.com/jeremyfa/yaml.js) library that we use is not among them, because the single newlines are preserved instead). This makes the generated docblocks hard to read, because the lines tend to be pretty long. This was generated by doing ` ksc -t java mac_os_resource_snd.ksy`:

```java
/**
 * Sound resources were introduced in Classic MacOS with the Sound Manager program. They can contain sound commands to generate sounds with given frequencies as well as sampled sound data. They are mostly found in resource forks, but can occasionally appear standalone or embedded in other files.
 * @see <a href="https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf">Source</a>
 */
public class MacOsResourceSnd extends KaitaiStruct {
```